.github/workflows/test.yml

```
@@ -133,6 +134,6 @@ jobs:
     - name: Print out version
       run: arm-none-eabi-gcc --version
     - name: Check the version matches
-      run: arm-none-eabi-gcc --version | grep -q 12.3.Rel1
+      run: arm-none-eabi-gcc --version | grep -q 13.2.Rel1
```

Looks like the output --version in this release hasn't capitalised `Rel`:
```
$ arm-none-eabi-gcc --version
arm-none-eabi-gcc (Arm GNU Toolchain 13.2.rel1 (Build arm-13.7)) 13.2.1 20231009
...

We can probably add the `-i` flag to grep to get this to pass.
```suggestion
      run: arm-none-eabi-gcc --version | grep -qi 13.2.Rel1
```
```